lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -351,12 +347,21 @@
                 createTeamsListView: function(options) {
                     var topic = options.topic,
                         collection = options.collection,
+                        self = this,
                         teamsView = new TopicTeamsView({
                             router: this.router,
                             context: this.context,
                             model: topic,
                             collection: collection,
-                            teamMemberships: this.teamMemberships,
+                            canUserCreateTeam: function () {
```

Comment:
I find it strange to have this definition defined here. I think the reason why this isn't defined in TopicTeamsView is that it doesn't have access to the myTeamsCollection, so maybe it should just be passed to it. Is there a use case for two different TopicTeamsViews having different implements of ``canUserCreateTeam``?